fenix/app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -151,6 +151,7 @@ private val darkColorPalette = FirefoxColors(
     borderAccent = PhotonColors.Violet40,
     borderDisabled = PhotonColors.LightGrey05A40,
     borderWarning = PhotonColors.Red40,
+    rippleEffect = PhotonColors.Black,
```

Comment:
Piggybacking on @MozillaNoah's comment, this doesn't seem like a color variable that is provided by the design system so I wouldn't want to see this here.

Reply:
I believe the correct answer here is to fallback to the MaterialTheme.

Reply:
The `RippleTheme` provided by `MaterialTheme` also seems to only darken the view, similar to what the default Compose ripple effect does.